core/src/main/java/org/jruby/ir/targets/JVMVisitor.java

```diff
@@ -495,6 +433,31 @@ private void jvmStoreLocal(Runnable source, Variable variable) {
         }
     }
 
+    private void genSetValue(LocalVariable localvariable) {
+        int depth = localvariable.getScopeDepth();
+        int location = localvariable.getLocation();
+
+        String baseName = p(DynamicScope.class);
+
+        if (depth == 0) {
+
```

Comment:
The newlines existed before the PR too. May be unnecessary with the simpler body, though.